DO NOT MERGE - Updating CLA and license (#23)
mulesoft/docs-site-antora-ui-sandbox: README.adoc

= We welcome your contributions

Before creating a pull request, please see

* xref:LICENSE.txt[LICENSE.txt]
* xref:SECURITY.adoc[SECURITY.adoc]
* xref:CODE_OF_CONDUCT.adoc[CODE_OF_CONDUCT.adoc]

> When you submit your pull request, you will be asked to sign a contributor license agreement (CLA) if we do not have one on file that is associated with your account.

Reviewer: We try to use simple present tense rather than future or past, when possible to retain meaning. Also, you can use a more informal style, such as...:
```suggestion
When you submit your pull request, you are asked to sign a contributor license agreement (CLA) if we don't have one on file for you.
```
